File: src/main/java/net/softbell/bsh/dto/view/admin/GroupNodeCardItemDto.java
package net.softbell.bsh.dto.view.admin;

import lombok.Getter;
import lombok.Setter;
import net.softbell.bsh.domain.entity.Node;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 그룹 노드 카드 아이템 DTO
 */
@Getter
@Setter
public class GroupNodeCardItemDto
{
	private Long nodeId;
	private String nodeName;
	private String alias;
	private String version;
	
	public GroupNodeCardItemDto(Node entity)
	{
		// Exception
		if (entity == null)
			return;
		
		// Convert
		this.nodeId = entity.getNodeId();
		this.nodeName = entity.getNodeName();
		this.alias = entity.getAlias();
		this.version = entity.getVersion();
	}
}

File: src/main/kotlin/net/softbell/bsh/dto/view/admin/GroupNodeCardItemDto.kt
package net.softbell.bsh.dto.view.admin

import net.softbell.bsh.domain.entity.Node

/**
 * @author : Bell(bell@softbell.net)
 * @description : 그룹 노드 카드 아이템 DTO
 */
class GroupNodeCardItemDto(entity: Node) {
    val nodeId: Long = entity.nodeId
    val nodeName: String = entity.nodeName
    val alias: String = entity.alias
    val version: String = entity.version
}